src/CattyTests/Bricks/RepeatUntilBrickTests.swift

```diff
@@ -51,25 +51,21 @@ final class RepeatUntilBrickTests: XCTestCase {
     }
 
     func testMutableCopy() {
-        brick.repeatCondition = Formula.init(double: 0)
```

Comment:
Please revert this test, since the previous test was a little bit more detailed, also testing false positives.